compiler/rustc_passes/messages.ftl

```diff
@@ -427,6 +427,10 @@ passes_link_section =
     .warn = {-passes_previously_accepted}
     .label = not a function or static
 
+passes_linkage =
+    attribute should be applied to a free function, impl method or static, foreign static
+    .label = not a free function, impl method or static, foreign static
```

Comment:
I propose

```suggestion
passes_linkage =
    attribute should be applied to a function defintion or static
    .label = not a function definition or static
```

I think static + foreign static covers all statics? The intuition for the function is that it must have a body, right? e.g. trait methods and extern fns are not allowed (see below).

Reply:
Thanks for the review! I came across some code that uses the linkage attribute for foreign functions, so I need to take some time to investigate it.

https://github.com/moturus/motor-os/blob/f55baab1ae3f9eef11302d86aa843b6995130b11/src/lib/moto-runtime/src/rt_api/process.rs#L32-L36

Reply:
Ahh, thanks. I've fixed it.